tests/upload-captures.sh

```diff
@@ -1,5 +1,10 @@
-for file in $(find tests/captures -name '*.png')
-do
-    raw_url=$(cat $file | base64 | gist --raw)
-    echo "https://base64service.herokuapp.com/decode?url=$raw_url&filename=$file"
-done
+files=$(find tests/captures -name '*.png')
+if $files; then
+    gem install gist
+
+    for file in $files
+    do
+        raw_url=$(cat $file | base64 | gist --raw)
+        echo "https://base64service.herokuapp.com/decode?url=$raw_url&filename=$file"
```

Comment:
I need to update it to put everything into one gist but it gets the job done as long as it doesn't go over the GitHub limit :p